[Bug]: Doesn't even compile....

### Summary

Doesn't compile, duh!

### Affected component

runtime/daemon

### Severity

S1 - workflow blocked

### Current behavior

==> Building release binary
   Compiling zeroclaw v0.1.7 (/home/micro/zeroclaw)
error[E0061]: this function takes 3 arguments but 2 arguments were supplied
    --> src/gateway/mod.rs:2176:15
     |
2176 |         match run_gateway_chat_with_tools(&state, &msg.content).await {
     |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^---------------------- argument #3 of type `std::option::Option<&str>` is missing
     |
note: function defined here
    --> src/gateway/mod.rs:1005:21
     |
1005 | pub(super) async fn run_gateway_chat_with_tools(
     |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^
...
1008 |     session_id: Option<&str>,
     |     ------------------------
help: provide the argument
     |
2176 |         match run_gateway_chat_with_tools(&state, &msg.content, /* std::option::Option<&str> */).await {
     |                                                               +++++++++++++++++++++++++++++++++

error[E0063]: missing field `quota_metadata` in initializer of `providers::traits::ChatResponse`
   --> src/providers/cursor.rs:233:12
    |
233 |         Ok(ChatResponse {
    |            ^^^^^^^^^^^^ missing `quota_metadata`

Some errors have detailed explanations: E0061, E0063.
For more information about an error, try `rustc --explain E0061`.
error: could not compile `zeroclaw` (lib) due to 2 previous errors


### Expected behavior

a flawless compilation that doesn't exit with errors!!!!

### Steps to reproduce

```bash
git clone https://github.com/zeroclaw-labs/zeroclaw.git
cd zeroclaw
cargo build --release --verbose
```

### Impact

devastating

### Logs / stack traces

```text

```

### ZeroClaw version

v0.1.7

### Rust version

rustc 1.93.1 (01f6ddf75 2026-02-11) (Arch Linux rust 1:1.93.1-1.1)

### Operating system

Linux cachyos-x8664 6.18.15-3-cachyos-lts

### Regression?

No, first-time setup

### Pre-flight checks

- [x] I reproduced this on the latest main branch or latest release.
- [x] I redacted secrets/tokens from logs.
- [x] I removed personal identifiers and replaced identity-specific data with neutral placeholders.

--- a/src/cron/store.rs
+++ b/src/cron/store.rs
@@ -337,7 +337,8 @@ pub fn record_run(
         )
         .context("Failed to prune cron run history")?;
 
-        tx.commit().context("Failed to commit cron run transaction")?;
+        tx.commit()
+            .context("Failed to commit cron run transaction")?;
         Ok(())
     })
 }

--- a/src/memory/sqlite.rs
+++ b/src/memory/sqlite.rs
@@ -337,8 +337,7 @@ impl SqliteMemory {
         category: Option<&str>,
         session_id: Option<&str>,
     ) -> anyhow::Result<Vec<(String, f32)>> {
-        let mut sql =
-            "SELECT id, embedding FROM memories WHERE embedding IS NOT NULL".to_string();
+        let mut sql = "SELECT id, embedding FROM memories WHERE embedding IS NOT NULL".to_string();
         let mut param_values: Vec<Box<dyn rusqlite::types::ToSql>> = Vec::new();
         let mut idx = 1;
 
@@ -500,13 +499,11 @@ impl Memory for SqliteMemory {
             let session_ref = session_id.as_deref();
 
             // FTS5 BM25 keyword search
-            let keyword_results =
-                Self::fts5_search(&conn, &query, limit * 2).unwrap_or_default();
+            let keyword_results = Self::fts5_search(&conn, &query, limit * 2).unwrap_or_default();
 
             // Vector similarity search (if embeddings available)
             let vector_results = if let Some(ref qe) = query_embedding {
-                Self::vector_search(&conn, qe, limit * 2, None, session_ref)
-                    .unwrap_or_default()
+                Self::vector_search(&conn, qe, limit * 2, None, session_ref).unwrap_or_default()
             } else {
                 Vec::new()
             };
@@ -604,11 +601,7 @@ impl Memory for SqliteMemory {
                         .iter()
                         .enumerate()
                         .map(|(i, _)| {
-                            format!(
-                                "(content LIKE ?{} OR key LIKE ?{})",
-                                i * 2 + 1,
-                                i * 2 + 2
-                            )
+                            format!("(content LIKE ?{} OR key LIKE ?{})", i * 2 + 1, i * 2 + 2)
                         })
                         .collect();
                     let where_clause = conditions.join(" OR ");

--- a/src/tools/git_operations.rs
+++ b/src/tools/git_operations.rs
@@ -735,10 +735,7 @@ mod tests {
         });
         let tool = GitOperationsTool::new(security, tmp.path().to_path_buf());
 
-        let result = tool
-            .execute(json!({"operation": "branch"}))
-            .await
-            .unwrap();
+        let result = tool.execute(json!({"operation": "branch"})).await.unwrap();
         // Branch listing must not be blocked by read-only autonomy
         let error_msg = result.error.as_deref().unwrap_or("");
         assert!(

